diff --git a/qrenderdoc/CMakeLists.txt b/qrenderdoc/CMakeLists.txt
index 22435f54e..28188be06 100644
--- a/qrenderdoc/CMakeLists.txt
+++ b/qrenderdoc/CMakeLists.txt
@@ -110,8 +110,6 @@ else()
     add_custom_command(OUTPUT RenderDoc.icns COMMAND touch RenderDoc.icns)
 endif()
 
-include(ExternalProject)
-
 # Need bison for swig
 find_package(BISON)
 
@@ -152,16 +150,6 @@ if(${CMAKE_VERSION} VERSION_GREATER "3.23")
     cmake_policy(SET CMP0135 NEW)
 endif()
 
-# Compile our custom SWIG that will do scoped/strong enum classes
-ExternalProject_Add(custom_swig
-    # using an URL to a zip directly so we don't clone the history etc
-    URL ${RENDERDOC_SWIG_PACKAGE}
-    BUILD_IN_SOURCE 1
-    CONFIGURE_COMMAND ${SET_SYSTEM_PATH_COMMAND} ./autogen.sh > /dev/null 2>&1
-    COMMAND CC=${SWIG_CONFIGURE_CC} CXX=${SWIG_CONFIGURE_CXX} CFLAGS=-fPIC CXXFLAGS=-fPIC ${SET_SYSTEM_PATH_COMMAND} ./configure --with-pcre=yes --prefix=${CMAKE_BINARY_DIR} > /dev/null
-    BUILD_COMMAND ${GENERATOR_MAKE} ${GENERATOR_MAKE_PARAMS} > /dev/null 2>&1
-    INSTALL_COMMAND ${GENERATOR_MAKE} install > /dev/null 2>&1)
-
 find_package(Shiboken2 QUIET)
 find_package(PySide2 QUIET)
 
